<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReplaceDialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>查找内容(&amp;N):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>73</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>替换为(&amp;P):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textLineEdit">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="replaceLineEdit">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>73</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>178</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>区分大小写(&amp;C)</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>-2</y>
     <width>111</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="findButton">
      <property name="text">
       <string>查找下一个(&amp;F)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="replaceButton">
      <property name="text">
       <string>替换(&amp;R)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="replaceAllButton">
      <property name="text">
       <string>全部替换(&amp;A)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
